### ISSUE TITLE
delata filter bug

### ISSUE THREAD

**Issue by wrowlands3** [dbuezas/lovelace-plotly-graph-card#202]
**Describe the bug**
delata filter results in error: [force_numeric,add,multiply,calibrate_linear,derivate,integrate,sliding_window_moving_average,median,exponential_moving_average,map_y_numbers,map_y,map_x,resample,store_var,fn,filter]
error shows up before a colon is added

**Screenshots**
![image](https://user-images.githubusercontent.com/86292646/214592737-f7393633-ba1f-4097-a968-36b9e49af512.png)

**yaml**
```yaml
type: custom:plotly-graph
entities:
  - entity: input_boolean.medicine
    filters:
      - map_y: 'state.state === "on" ? 1 : 0'
      - sliding_window_moving_average:
          window_size: 1
  - entity: input_boolean.medicine
    filters:
      - map_y: 'state.state === "on" ? 1 : 0'
  - entity: input_boolean.medicine
    filters:
      - map_y: 'state.state === "on" ? 1 : 0'
      - delata
hours_to_show: 0.5
refresh_interval: 10
```
**Additional context**
Attempting to use the filters so that a change of state (in this case a boolean) creates a single vertical line, to use as a visual timestamp overlay on other graphs.

**Comment by dbuezas** [comment]
oh, the readme had a typo. The filter name is `delta`. Not del~a~ta. Readme is fixed now.

**Comment by dbuezas** [comment]
And that filter is not released yet. Sorry for the confusion, I should use a separate branch until I make a release

**Comment by dbuezas** [comment]
available in v3.0.0-beta

**Comment by dbuezas** [comment]
Closed by the release of v3.0.0
